File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        cnt = [0]*26
        for i in range(len(s)):
            cnt[ord(s[i])-ord('a')] += 1
            cnt[ord(t[i])-ord('a')] -= 1
        ans = 0
        for i in range(26):
            ans += max(0, cnt[i])
        return ans

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] cnt = new int[26];
        for(int i=0; i<s.length(); i++){
            cnt[s.charAt(i)-'a']++;
            cnt[t.charAt(i)-'a']--;
        }
        int ans = 0;
        for(int i=0; i<26; i++) ans += Math.max(0, cnt[i]);
        return ans;
    }
}